### ISSUE TITLE
Assertion fail on bone position update

### ISSUE THREAD

**Issue by Foghrye4** [minetest/minetest#4635]
Using command set_bone_position with arguments out of range in Lua side cause assertion fail on C++ side.
Minetest version: 0.4.14-dev-7da3c36
(because of another bug current head version of Minetest will not work)
Example of such command:
`player:set_bone_position("Head", {x=0,y=1,z=0}, {x=1e100,y=0,z=1e100})`
Error log: http://pastebin.com/XSDeGYMg

Original command causing problem:
```
local look_horizontal = player:get_look_horizontal()
local look_vertical = player:get_look_vertical()
player:set_bone_position("Head", {x=0,y=1,z=0}, {x=look_vertical*180/3.14159,y=0,z=90-look_horizontal*180/3.14159})
```

**Comment by sfan5** [comment]
And why is this a bug?

**Comment by Foghrye4** [comment]
sfan5, you are right. I just wish where would be a code, which will check Lua input values and raise Lua error before assertion check. Because currently no error info in "debug.txt" after program termination.

**Comment by Foghrye4** [comment]
Solved by #4657.
